renku/data/pre-commit.sh

```diff
@@ -38,7 +38,7 @@ if [ ${#MODIFIED_FILES[@]} -ne 0 ] || [ ${#ADDED_FILES[@]} -ne 0 ]; then
 fi
 
 if [ ${#MODIFIED_FILES[@]} -ne 0 ] ; then
-  MODIFIED_OUTPUTS=$(renku show outputs "${MODIFIED_FILES[@]}")
+  MODIFIED_OUTPUTS=$(renku --disable-version-check show outputs "${MODIFIED_FILES[@]}")
```

Comment:
you could set the `RENKU_DISABLE_VERSION_CHECK` environment variable to avoid a regression in case more commands are added here and we forget about the flag?

Reply:
I didn't know that env var exists, thanks for pointing it out. Makes more sense